api/src/main/java/org/openmrs/util/databasechange/MigrateAllergiesChangeSet.java

```diff
@@ -162,7 +162,10 @@ private Integer getConceptByGlobalProperty(Database database, String globalPrope
 		if (rs.next()) {
 			String uuid = rs.getString("property_value");
 			
-			rs = stmt.executeQuery("SELECT concept_id FROM concept WHERE uuid = '" + uuid + "'");
+//			rs = stmt.executeQuery("SELECT concept_id FROM concept WHERE uuid = '" + uuid + "'");
+			PreparedStatement conceptStmt = connection.prepareStatement("SELECT concept_id FROM concept WHERE uuid = ?");
```

Comment:
Fix for the SQL Injection Bug reported in Project Part 3 -> Fortify Analysis Defects -> (2) SQL Injection (A1: Injection)